cloudigrade/account/tasks.py

```diff
@@ -228,3 +235,54 @@ def run_inspection_cluster(messages, cloud='aws'):
         cluster=settings.HOUNDIGRADE_ECS_CLUSTER_NAME,
         taskDefinition=result['taskDefinition']['taskDefinitionArn'],
     )
+
+
+def persist_aws_inspection_cluster_results(inspection_result):
+    """
+    Persist the aws houndigrade inspection result.
+
+    Args:
+        inspection_result (dict): A dict containing houndigrade results
+    Returns:
+        None
+    """
+    RHEL_TAG = ImageTag.objects.filter(description='rhel').first()
+    results = inspection_result.get('results', [])
+    for image_id, image_json in results.items():
+        ami = AwsMachineImage.objects.filter(ec2_ami_id=image_id).first()
+        if ami is not None:
+            ami.tags.clear()
+            rhel_found = any([attribute.get('rhel_found')
+                              for disk_json in list(image_json.values())
+                              for attribute in disk_json.values()])
+            if rhel_found:
+                ami.tags.add(RHEL_TAG)
+            # Add image inspection JSON
+            ami.inspection_json = json.dumps(image_json)
+            ami.save()
+        else:
+            logger.error(_('AwsMachineImage %s is not found.' % image_id))
```

Comment:
We've generally been using .format for our strings, in addition this won't actually work with gettext since it'll try to look it up with the image_id in the string. Try changing it to:
```
logger.error(_('AwsMachineImage "{0}" is not found.').format(image_id))
```